### PR TITLE
Reusable commands

### LINKED ISSUE #22: reuse commands in symfony
cli: https://github.com/rokka-io/rokka-client-php-cli/pull/31
* [x] adjust service names in command.xml and restricted_commands.xml so that they are fit to be loaded in the symfony bundle (proper namespace prefix)
* [x] have optional constructor argument to specify a name prefix

bundle https://github.com/rokka-io/rokka-client-bundle/pull/7
* [x] provide configuration class as service in symfony bundle
* [x] add the prefix argument when we load the services in the rokka bundle

### PR DESCRIPTION
**Pull request by dbu**
fix #22

### CODE REVIEW

File: src/Command/DynamicMetadata/ImageDynamicMetadataDeleteSubjectAreaCommand.php [hunk 1]
```diff
@@ -14,7 +14,7 @@ class ImageDynamicMetadataDeleteSubjectAreaCommand extends BaseRokkaCliCommand
     protected function configure()
     {
         $this
-            ->setName('image:delete-subjectarea')
+            ->setName($this->namePrefix.'image:delete-subjectarea')
```
Comment (thePanz) [inline src/Command/DynamicMetadata/ImageDynamicMetadataDeleteSubjectAreaCommand.php]: Can't we override the `BaseRokkaCliCommand::setName()` to have the prefix appended by default?
Do you see any issue with that?

**Comment by thePanz** [comment]
Thanks @dbu! Looks good to me, could you check my comment above?

**Comment by dbu** [comment]
excellent point, no idea why i did it as complicated. fixed that.

**Comment by dbu** [comment]
time to release 1.2.0? (if you want to do it, please build the phar locally and check that everything really still works - i did not build the phar, only run the bin script locally)

**Comment by thePanz** [comment]
Version 1.2.1 has been released! :)

**Pull request merged**